dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-gitlab-dynamic/src/dynamic/index.ts

```diff
@@ -4,19 +4,15 @@ import { GitlabDiscoveryEntityProvider } from '@backstage/plugin-catalog-backend
 export const dynamicPluginInstaller: BackendDynamicPluginInstaller = {
   kind: 'legacy',
   async catalog(builder, env) {
-    if (env.config.getOptionalBoolean('enabled.gitlab') || false) {
-      builder.addEntityProvider(
-        ...GitlabDiscoveryEntityProvider.fromConfig(env.config, {
-          logger: env.logger,
-          schedule: env.scheduler.createScheduledTaskRunner({
-            frequency: { minutes: 30 },
-            timeout: { minutes: 3 },
-          }),
-          scheduler: env.scheduler,
+    builder.addEntityProvider(
+      ...GitlabDiscoveryEntityProvider.fromConfig(env.config, {
+        logger: env.logger,
+        schedule: env.scheduler.createScheduledTaskRunner({
+          frequency: { minutes: 30 },
+          timeout: { minutes: 3 },
         }),
-      );
-    } else {
-      env.logger.info('Gitlab plugin is disabled');
-    }
+        scheduler: env.scheduler,
```

Comment:
Similarly to the github provider above, [gitlab](https://github.com/backstage/backstage/blob/438f7f87ac61095ae700e085a375c41cf16f5c58/plugins/catalog-backend-module-gitlab/src/providers/GitlabDiscoveryEntityProvider.ts#L82-L84) also has this issue upstream